src/main/java/gregtech/loaders/recipe/VanillaStandardRecipes.java

```diff
@@ -350,18 +350,20 @@ private static void woodRecipes() {
                 .chancedOutput(dust, Wood, 8000, 680)
                 .buildAndRegister();
 
-        LATHE_RECIPES.recipeBuilder()
-                .input(plank, Wood)
-                .output(stick, Wood, 2)
-                .duration(10).EUt(VA[ULV])
-                .buildAndRegister();
+        if (ConfigHolder.recipes.WoodCraftingNumber > 0) {
```

Comment:
I though this configuration was supposed to modify only hand crafted recipes not machine processed.